helm/g8s-oauth2-proxy-chart/templates/deployment.yaml

```diff
@@ -16,14 +16,18 @@ spec:
         k8s-app: oauth2-proxy
     spec:
       containers:
-        # Use bitly fork from https://github.com/giantswarm/oauth2_proxy
-      - image: quay.io/giantswarm/oauth2_proxy:38072cbaa6abfc6e9f647c1ee0e4c7afa5ba194e
+        # Use bitly fork from https://github.com/pusher/oauth2_proxy/tree/kubernetes,
+        # which supports OICD token renewal and has ca certs installed
+      - image: {{ .Values.Installation.V1.Registry.Domain }}/giantswarm/oauth2_proxy:kubernetes-3-ca-certs
```

Comment:
The new image (with same name) with supported feats. Pull image from aliyun when in china.